src/Utility/getCroppedCanvas.js

```diff
@@ -0,0 +1,78 @@
+export function getCroppedCanvas(ctx) {
+    const {width, height } = ctx.canvas;
+    if (!width && !height) { return ctx.canvas; }
+
+    const imgData = ctx.getImageData(0, 0, width, height);
+    const data = new Uint32Array(imgData.data.buffer);
+    data[0] = 0; // TODO: pointerout handler colors pixel at 0 0 black, so we set it back to transparent;
+                 // I tracked the issue to detectBlockedCanvasImageExtraction function;
```

Comment:
Puh, sorry for leaving you to such badly documented code. I think `detectBlockedCanvasImageExtraction` might be responsible, but that's @baltpeter's code, so I'm pinging him.

Reply:
We were indeed not correctly restoring the old pixel. Thanks for pointing that out, @merelj. I opened #137 to fix that.

Reply:
thank you for fixing it so quickly 👍